src/cli/cli.cpp

```diff
@@ -4785,7 +4831,7 @@ template <> otError Interpreter::Process<Cmd("neighbor")>(Arg aArgs[])
      * Done
      * @endcode
      * @par
-     * Print the connection time and age of neighbors. Info per neighbor:
+     * Prints the connection time and age of neighbors. Information per neighbor:
      * - RLOC16
      * - Extended MAC address
```

Comment:
Since you're editing this one, can you update the list so that the values are exactly the same as the table headers?  Plus the description of each column shouldn't be in parentheses